tests/test_ramp_fitting.py

```diff
@@ -614,12 +617,11 @@ def test_one_group_ramp_suppressed_two_integrations():
 
     # Check slopes information
     sdata, sdq, svp, svr, serr = slopes
-    sat, dnu = dqflags["SATURATED"], dqflags["DO_NOT_USE"]
 
     check = np.array([[1.0000001, 1.0000001, 1.0000002]])
     np.testing.assert_allclose(sdata, check, tol)
 
-    check = np.array([[sat, sat, 0]])
+    check = np.array([[GOOD, GOOD, GOOD]])
     np.testing.assert_allclose(sdq, check, tol)
```

Comment:
Comparing with the previous version, should this be np.array([[SAT, SAT, GOOD]]) or is this a different test ? This also applies to another later test.

Reply:
Since the way the SATURATED flag gets set the expected value for the SATURATED flag.  As the test is written, with the new behavior of setting the SATURATED flag in the rate file, the new expected output is `[GOOD, GOOD, GOOD]`.